pkg/preflight/utils.go

```diff
@@ -33,7 +33,8 @@ func IsAWS() bool {
 func RequiresCheck() bool {
 	return Instance().ProviderName() == string(cloudops.AWS) ||
 		Instance().ProviderName() == string(cloudops.Vsphere) ||
-		Instance().ProviderName() == string(cloudops.Pure)
+		Instance().ProviderName() == string(cloudops.Pure) ||
+		Instance().ProviderName() == string(cloudops.Azure)
 }
 
 // RunningOnCloud checks whether portworx is running on cloud
```

Comment:
Not related to your change but we should add IsGKE(), IsAWS(), IsAzure() under the RunningOnCloud() function.

Reply:
Will add it to the next checkin...